app/models/server_role.rb

```diff
@@ -2,8 +2,12 @@ class ServerRole < ApplicationRecord
   has_many :assigned_server_roles
   has_many :miq_servers, :through => :assigned_server_roles
 
-  validates_presence_of     :name
-  validates_uniqueness_of   :name
+  validates :name, :presence => true, :uniqueness => true
+
+  scope :all_names,      -> { order(:name).pluck(:name) }
```

Comment:
I don't think we should make this a scope as `#pluck` returns an array.

I think scopes are meant to be chained with additional AR methods. You won't be able to do that in this case.

Reply:
👍 Good point.  Updated accordingly